File: FlightMobileApp/app/src/main/java/services/joystick/Command.kt
package services.joystick

import com.google.gson.annotations.SerializedName

data class Command(
    @SerializedName("aileron") val Aileron: Float,
    @SerializedName("elevator") val Elevator: Float,
    @SerializedName("throttle") val Throttle: Float,
    @SerializedName("rudder") val Rudder: Float
)

File: FlightMobileServer/FlightMobileWeb/Model/Command.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace FlightMobileWeb.Model
{
    public class Command
    {
        public Command()
        {
            Aileron = -10;
            Rudder = -10;
            Elevator = -10;
            Throttle = -10;
        }

        [JsonProperty, JsonPropertyName("aileron")]
        public double Aileron { get; set; }

        [JsonProperty, JsonPropertyName("rudder")]
        public double Rudder { get; set; }

        [JsonProperty, JsonPropertyName("elevator")]
        public double Elevator { get; set; }

        [JsonProperty, JsonPropertyName("throttle")]
        public double Throttle { get; set; }
    }
}
